plugins/scaffolder-backend/src/service/helpers.ts

```diff
@@ -107,3 +107,35 @@ export async function findTemplate(options: {
 
   return template as TemplateEntityV1beta3;
 }
+
+/**
+ * Takes a single unknown parameter and makes sure that it's a single string or
+ * an array of strings, and returns as an array.
+ */
+export function parseStringsParam(
+  param: unknown,
+  paramName: string,
+): string[] | undefined {
+  if (param === undefined) {
+    return undefined;
+  }
+
+  const array = [param].flat();
+  if (array.some(p => typeof p !== 'string')) {
+    throw new InputError(`Invalid ${paramName}, not a string`);
```

Comment:
It seems not fully correct error message. 
As the JsDoc saying that the array of strings is also a valid param.

Reply:
Sorry but what do you mean by this? 🤔 This function is copied from catalog utilities and works the same way there

Reply:
I mean that if `paramName` is an array of strings, it's still valid.

/**
 * Takes a single unknown parameter and makes sure that it's a single string or
 * an array of strings, and returns as an array.
 */

`throw new InputError(`Invalid ${paramName}, not a string or array of strings`);` would be more correct.

Could be also incorrect in catalog then 😅 

If I'm as a user will read the error and not check the code, I would have no clue that I can also provide the array of strings